source/03-specifications/03-counter-report-common-attributes-and-elements.rst

```diff
@@ -371,12 +371,12 @@ Table 3.r (below): List of Metric_Types for Searches
 
 This group of Metric_Types represents activities where content items were retrieved (Requests) or information about a content item (e.g. an abstract) was examined (Investigations). Any user activity that can be attributed to a content item will be considered an Investigation including downloading or viewing the item. Requests are limited to user activity related to retrieving or viewing the content item itself. The figure below provides a graphical representation of the relationship between Investigations and Requests.
 
-.. figure:: ../_static/img/3e-investigations-and-requests.png
+.. figure:: ../_static/img/Figure-3e.png
```

Comment:
I find the Investigations part in the old image easier to understand, especially:

- The terms are ambiguous without the "user action" part. Only a user action that results for example in a metdata record to be shown is an Investigation, but not a metadata record for example shown in a result list.
- The outgoing link actions are missing. In my opinion these are important examples that are not obvious.